core-bundle/src/EventListener/DataContainer/PageUrlListener.php

```diff
@@ -101,7 +103,9 @@ public function generateAlias(string $value, DataContainer $dc): string
      */
     public function validateUrlPrefix(string $value, DataContainer $dc): string
     {
-        if ('root' !== $dc->activeRecord->type || $dc->activeRecord->urlPrefix === $value) {
+        $currentRecord = $dc->getCurrentRecord();
+
+        if (null === $currentRecord || 'root' !== $currentRecord['type'] || $currentRecord['urlPrefix'] === $value) {
```

Comment:
You have added a `null` check here but not in line 84. Why?

Reply:
No specific reason, we might as well remove it but it doesn't matter at all.

Reply:
```suggestion
        if ('root' !== $currentRecord['type'] ?? null || $currentRecord['urlPrefix'] === $value) {
```